harbors/schema.py

```diff
@@ -0,0 +1,59 @@
+import graphene
+import graphql_geojson
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+
+from .models import AvailabilityLevel, BoatType, Harbor
+
+
+class AvailabilityLevelType(DjangoObjectType):
+    class Meta:
+        model = AvailabilityLevel
+
+    title = graphene.String()
+    description = graphene.String()
+
+
+class BoatTypeType(DjangoObjectType):
+    class Meta:
+        model = BoatType
+
+    name = graphene.String()
+
+
+class HarborType(graphql_geojson.GeoJSONType):
+    class Meta:
+        model = Harbor
+        filter_fields = [
+            "mooring",
+            "electricity",
+            "water",
+            "waste_collection",
+            "gate",
+            "lighting",
+            "suitable_boat_types",
+            "maximum_width",
+            "maximum_length",
+        ]
+        geojson_field = "location"
+        interfaces = (relay.Node,)
+
+    name = graphene.String()
+    street_address = graphene.String()
+
+
+class Query(object):
```

Comment:
Python 2 syntax. In Python 3 `object` is the default base class, so you can just do 
```python
class Query:
```